package-lock.json

```diff
@@ -40,35 +40,24 @@
         "react-dom": "^18.2.0"
       },
       "devDependencies": {
-        "@babel/core": "^7.20.2",
-        "@babel/preset-env": "^7.20.2",
-        "@babel/preset-react": "^7.18.6",
         "@cloudoperators/juno-config": "*",
         "@svgr/core": "^8.0.0",
         "@svgr/plugin-jsx": "^8.0.0",
         "@tanstack/react-query": "5.36.2",
-        "@testing-library/dom": "^10.0.0",
-        "@testing-library/jest-dom": "^6.0.0",
-        "@testing-library/react": "^16.0.0",
-        "@testing-library/user-event": "^14.4.3",
-        "assert": "^2.0.0",
+        "@testing-library/react": "^16.0.1",
         "autoprefixer": "^10.4.2",
-        "babel-jest": "^29.3.1",
-        "babel-plugin-transform-import-meta": "^2.2.0",
-        "esbuild": "^0.20.1",
         "interweave": "^13.1.0",
-        "jest": "^29.3.1",
-        "jest-environment-jsdom": "^29.3.1",
         "jsdoc": "^4.0.2",
+        "jsdom": "^25.0.1",
         "luxon": "^3.0.0",
         "postcss": "^8.4.31",
-        "postcss-url": "^10.1.3",
         "prop-types": "^15.8.1",
         "react-markdown": "^9.0.0",
         "react-test-renderer": "^18.2.0",
         "sass": "^1.77.5",
-        "shadow-dom-testing-library": "^1.7.1",
+        "shadow-dom-testing-library": "^1.11.3",
         "tailwindcss": "^3.3.1",
+        "vitest": "^2.1.1",
```

Comment:
One question: Why is vitest listed under devDependencies in package-lock.json but not in package.json? It's not in the main package.json or included in this PR.

Reply:
Hi Hoda! You might have missed it, but it's in the package.json. Check the link from my branch :)
https://github.com/cloudoperators/juno/blob/artie-doop-vite/apps/doop/package.json#L31